### PR TITLE
bug 1707665 adding matchLabels

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1707665

@sunzhaohua2, will you PTAL?

### CODE REVIEW

**Comment by sunzhaohua2** [comment]
Between line 139 and 140 also should add machine labels :
```
    machine.openshift.io/cluster-api-machine-role: <machine_label> 
    machine.openshift.io/cluster-api-machine-type: <machine_label> 
```

**Comment by kalexand-rh** [comment]
@sunzhaohua2, is this right now?

File: modules/machineset-creating.adoc [hunk 1]
```diff
@@ -137,6 +139,8 @@ spec:
       machine.openshift.io/cluster-api-machineset: <cluster_name>-<machine_label>-<AWS-availability-zone>
```
Comment (sunzhaohua2) [inline modules/machineset-creating.adoc]: I am so sorry, it should be added above this line.
```
    machine.openshift.io/cluster-api-machine-role: <machine_label> 
    machine.openshift.io/cluster-api-machine-type: <machine_label>
```

Comment (kalexand-rh) [reply modules/machineset-creating.adoc]: @sunzhaohua2, no worries! I've relocated the lines.

**Comment by sunzhaohua2** [comment]
LGTM, thanks!

**Comment by kalexand-rh** [comment]
Thanks! Merging.

**Pull request merged**